packages/frontend/app/components/reports/subject/competency.gjs

```diff
@@ -49,6 +49,41 @@ export default class ReportsSubjectCompetencyComponent extends Component {
     return !this.args.school;
   }
 
+  async getResultsForSessionType(sessionTypeId, filters) {
+    filters.push(`id: ${sessionTypeId}`);
+
+    const results = await this.graphql.find(
+      'sessionTypes',
+      filters,
+      'school { competencies { id, title, school { title } } }',
+    );
```

Comment:
this gives you all the competencies defined the school that the given session type is defined in.
which yields the same list of all competencies for each session type defined in the same given school.

this is not the data we're after. you'd have to climb through session-type > session > session objective > course-objective > program-year objective > competency instead.

i'm not sure if this is even viable.
but this implementation isn't yielding the correct results.